content/blog/2022-07-26-iterative-studio-model-registry.md

```diff
@@ -0,0 +1,185 @@
+solution.
+
+**We built the Iterative Studio Model Registry to solve these problems.**
+
+Iterative Studio Model Registry enables ML teams to collaborate on models by
+providing model organization, discovery, versioning, lineage (tracing the origin
+of the model), and the ability to manage deployment statuses such as,
+development, staging, and production across multiple projects.
+
+## Utilize your existing Git infrastructure
+
+Iterative Studio Model Registry is built on top of Git, which means:
+
+- You can reuse your existing Git infrastructure to manage ML models together
+  with code, data, experiment pipelines, and deployment statuses.
+- You can use GitOps for model deployment, and trigger model deployment from
+  Iterative Studio, which you can also use to run your ML experiments.
+- Your DS/ML folks and Software/DevOps folks can work together more easily,
+  because they utilize the same tools and infrastructure.
+
+## Open MLOps
+
+A core philosophy at Iterative is open MLOps - we build tools that work with
+your infrastructure. Our toolstack is modular, so you can build your model
+registry on top of your existing cloud and DevOps infrastructure and avoid
+vendor lock-in.
+
+Under the hood, Iterative Studio Model Registry uses Iterative’s open-source
+Git-based tools [GTO] and [MLEM].
+
+- [GTO] enables [semantic versioning][semver] and stage transitions of artifacts
+  using metadata files and Git tags.
+- [MLEM] saves ML models and extracts model metadata including framework,
+  methods, input / output data schema, and requirements.
+
+![Iterative toolstack is modular](/uploads/images/2022-07-26/modular-toolstack.png)
+
+## UI of your choice
+
+Iterative Studio Model Registry can be used through your favorite interface,
+whether you’re a data scientist who likes APIs,a manager who prefers a web UI,
+or a DevOps engineer who works best with the command line interface (CLI).
+
+## Models can reside anywhere
+
+Save your model files wherever works best for you, whether it’s in S3, GCP, or
+any other of your remote (or local) storages. Then, add them to the model
+registry in a non-intrusive, no-code fashion **without modifying your ML
+training code**. This saves you hours of valuable time.
+
+## Create model versions and stages from any Git commit
+
+For registering versions, select the commit and provide the version number. To
+assign stages, select the version and provide the stage name. It is as simple as
+that.
+
+## Collaborate across multiple projects
+
+A central dashboard of all your models facilitates transparency and discovery
+across every project by your whole team.
+
+![Models are organized in a central dashboard](/uploads/images/2022-07-26/models-dashboard.png)
+
+And on the model details page, you’ll find that information about the model is
+automatically detected and its history tracked.
+
+![All models have separate model detail pages](/uploads/images/2022-07-26/model-details-page.png)
```

Comment:
This image needs to be corrected to remove the extra tags.  Let me know if you need me to do it for you.

Reply:
Fixed.